src/stan-users-guide/simple_linear_regression/output/.gitignore

```diff
@@ -0,0 +1,5 @@
+#OSX
```

Comment:
I don't think .gitignore should be in subdirectories, just the top-level directory.

Reply:
output/.gitignore ignores .csv and .txt files that are generated by running cmdstanR and cmdstanPy. Having it higher would filter below that or I guess I could put paths in but that gets ugly too. 

In docs/.gitignore gets added

```
#output files for footnotes
src/stan-users-guide/*/output/*.txt
src/stan-users-guide/*/output/*.csv
```
If you are ok with that solution then I'll do it that way. 
I don't have a better idea.

Reply:
Cool---didn't know you could even put .gitignore lower down.  I don't think we do it anywhere else, but it makes sense here.  

Where are these models going to go?  Will they be under src/stan-users-guide or under something like src/stan or src/footnotes (I really really don't like "footnotes" as the name of the directory for example models---it makes it sound very secondary)?  The bibtex is on the outside.